ratlas_report-writing-vignette.Rproj

```diff
@@ -0,0 +1,17 @@
+Version: 1.0
```

Comment:
No need to create another .Rproj. Having multiple project files will mess with things. Just use the normal `ratlas.Rproj`